config/routes.rb

```diff
@@ -18,4 +24,8 @@
 
   resources :cart_items, only: [:create, :destroy]
   get "/cart", to: "cart_items#index"
+  
+  match '/404', to: 'errors#file_not_found', via: :all
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.